www/source/demo/packaging-system/steps/6.html.slim

```diff
@@ -0,0 +1,60 @@
+---
+title: Habitat Packaging System Demo - Step 6
+layout: layout
+classes: demo_step
+---
+#demo-step.demo--page
+  .row
+    .columns.large-12.demo-step--content
+      .demo-step--indicators
+        a.demo-step--indicator.packaging.active href="/demo/packaging-system/steps/1"
+        a.demo-step--indicator.packaging.active href="/demo/packaging-system/steps/2"
+        a.demo-step--indicator.packaging.active href="/demo/packaging-system/steps/3"
+        a.demo-step--indicator.packaging.active href="/demo/packaging-system/steps/4"
+        a.demo-step--indicator.packaging.active href="/demo/packaging-system/steps/5"
+        a.demo-step--indicator.packaging.active href="/demo/packaging-system/steps/6"
+      .row
+        .columns.large-12
+          h1.h4="Step 6: Run the application"
+        .columns.large-6
+          p
+            ' Now that the build is complete, let's test it out by running the package. Two common ways of running a package include:
+          ul
+            li Start services directly from the Studio* using the Habitat Supervisor, or
+            li Export the .hart to a Docker image and use <em>docker run</em>.
+          p
+            ' We've mentioned the benefits of Habitat's export capabilities, so let's see how
+            ' how that works now. With a single command, you will have exported the .hart file as a Docker image.
+          = code(:shell) do
+            | [1][default:/src:0]# hab pkg export docker ./results/<path to .hart>
```

Comment:
does this need the [1]? Also, the text is too long and falls out of the text box so you can't see all of it

![image](https://user-images.githubusercontent.com/719646/34234899-f66f0a1e-e5a2-11e7-9550-f2351c265c87.png)